### DISCUSSION TITLE
Running locally with `{{parentOrg}}` in config and it shows up as empty

### DISCUSSION THREAD

**Discussion by nitrocode** [renovatebot/renovate#30851] [Request Help]
### What would you like help with?

None

### How are you running Renovate?

None

### Please tell us more about your question or problem

I'm running renovate locally from a container

```bash
✗ ls -la | grep git/
drwxr-xr-x  14 user staff    448 Aug 17 11:57 .git/
✗ git config --get remote.origin.url
git@github.com:org/repo.git
✗ export GITHUB_TOKEN=$(gh config get -h github.com oauth_token)
✗ docker run --rm -ti \
  -e LOG_LEVEL=debug \
  -e GITHUB_COM_TOKEN="$GITHUB_TOKEN" \
  -v /tmp:/tmp \
  -v $(PWD):/usr/src/app \
  docker.io/renovate/renovate:full renovate --platform=local | tee renovate-my-token.log
```

(I also tried adding `-e RENVOATE_PARENT_ORG="org"`)

Using config containing a `depNameTemplate` of `"{{parentOrg}}/{{depName}}"`.

```json
  "customManagers": [
    {
      "_": "// ...",
      "depNameTemplate": "{{parentOrg}}/{{depName}}"
    }
  ],
```

How come `parentOrg` returns an empty value?

Is there a way to fill it with an environment variable?

The limitation isn't shown in these docs
- https://docs.renovatebot.com/modules/platform/local/
- https://docs.renovatebot.com/templates/

### Logs (if relevant)

The result is the following in the logs

The logs do not show any output related to an empty `parentOrg` other than showing the matched dependency without the org in the `packageName`

```json
             "deps": [
               {
                 "depName": "/mypackage",
                 "_": "// ...",
                 "packageName": "/mypackage"
               }
             ],
```

---

Seems like parentOrg was empty when I used it in renovate.

There are other inputs too. Unsure of the differences.

https://docs.renovatebot.com/templates/

- `parentOrg`
- `sourceRepoOrg`
- `topLevelOrg`

### Code references

https://github.com/renovatebot/renovate/blob/1895d0bfbf22af354907ef5e291310d5efbf4416/lib/workers/global/index.ts#L33-L43

https://github.com/renovatebot/renovate/blob/1895d0bfbf22af354907ef5e291310d5efbf4416/lib/workers/global/index.ts#L183-L190

https://github.com/renovatebot/renovate/blob/1895d0bfbf22af354907ef5e291310d5efbf4416/lib/workers/global/index.ts#L168-L171

**Comment by RahulGautamSingh** [comment]
From the docs: https://docs.renovatebot.com/configuration-options/#depnametemplate

>  It will be compiled using Handlebars and the regex groups result.

You can only use the named groups which are present in the regex of `matchStrings`. The values from config aren't used here.

**Reply by nitrocode** [reply]
Oh I see.

Then I am left with hard coding the org in the config which is unfortunate because I have multiple orgs to administer the renovate configurations.

Unless is there an alternative solution?

If not, may this be a feature request? 

Thank you!
